lib/sudoers/src/test/mod.rs

```diff
@@ -153,15 +153,16 @@ fn permission_test() {
     pass!(["user ALL=/bin/hello  arg"], "user" => root(), "server"; "/bin/hello arg");
     pass!(["user ALL=/bin/hello arg"], "user" => root(), "server"; "/bin/hello  arg");
     FAIL!(["user ALL=/bin/hello arg"], "user" => root(), "server"; "/bin/hello boo");
-    pass!(["user ALL=/bin/hello a*g"], "user" => root(), "server"; "/bin/hello  aaaarg");
-    FAIL!(["user ALL=/bin/hello a*g"], "user" => root(), "server"; "/bin/hello boo");
+    // several test cases with globbing in the arguments are explicitly not supported by sudo-rs
+    //pass!(["user ALL=/bin/hello a*g"], "user" => root(), "server"; "/bin/hello  aaaarg");
```

Comment:
One other option is to turn them into failing test cases instead of passing ones, of course. In any case I would like to keep them in in some form for now (and I want to rework this particular test function as well, as it is a bit heavy on the macro-usage)